ccm_web/client/src/components/CustomTable.tsx

```diff
@@ -51,7 +51,7 @@ function CustomTable<T extends TableEntity> (props: TableProps<T>): JSX.Element
         <TableBody>
           {tableRows.slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage).map((row) => {
             return (
-              <TableRow hover role="checkbox" tabIndex={-1} key={row.rowNumber}>
+              <TableRow hover key={row.rowNumber}>

```

Comment:
I have tested the confirmation screen problem that is described in the issue, seems to the fix is implemented as per the recommendation.
But one thing I am not clear is, the table content is not read by screen readers in case of success/failure. Is that a separate not relavent to this fix?

Reply:
Yeah, I think I want to resolve that as part of #258. We need to be smarter with our use of `aria-live`.

Reply:
OK, in that case, this issues is resolved to me